INSTALL.md

```diff
@@ -40,35 +40,29 @@ Dangerzone is available for:
 
 Add our repository following these instructions:
 
-Install `curl`, if it's not already available:
+Install `gpg`, if it's not already available:
 
 ```sh
-sudo apt-get install -y curl
+sudo apt-get install -y gpg ca-certificates
```

Comment:
Nitpick, but can we use the `apt` instead of `apt-get`? It's supposedly higher level (according to the man pages).

Reply:
Also, I was trying to check if these packages weren't already default in debian, but apparently that's not an easy task without Debian itself. @legoktm do you know if these are default in debian?

Reply:
Thanks! In that case I'd argue for removing this line. But can try to test it via docker as well, just to be sure.

Reply:
I've originally added this line because, as Kunal mentioned, these two packages are not part of the base system packages. This means that on a stripped down container image, they do not exist (tested).

Worse, if just `gpg` exists, but not `ca-certificates`, this will lead to a cryptic error that took me a bit to debug, when we download keys via the `hkps` protocol.

However, I agree with the sentiment that Dangerzone is a desktop application, therefore almost all users will have `ca-certificates` installed, and `gpg` most probably. So I'll remove this line to shorten the instructions.

Thanks again for the feedback :-)